@ershockley , e77dd0e addressed your comments. It's just `from admix.interfaces.rucio_api import RucioAPI` that is still causing issues. I also don't like the way this is imported but moving it somewhere else causes this error on import admix:
```python
(strax) joran@DESKTOP-F4PI41P:~/google_drive/ubuntu-windows/software/admix$ pytest
============================================================================================================ test session starts ============================================================================================================
platform linux -- Python 3.8.5, pytest-6.1.1, py-1.9.0, pluggy-0.13.1
rootdir: /mnt/d/Google_Drive/PhD-master/ubuntu-storage/ubuntu-windows/software/admix
plugins: hypothesis-5.37.1, cov-2.10.1
collected 0 items / 1 error

================================================================================================================== ERRORS ===================================================================================================================
___________________________________________________________________________________________________ ERROR collecting tests/test_admix.py ____________________________________________________________________________________________________
tests/test_admix.py:6: in <module>
    import admix
admix/__init__.py:11: in <module>
    from admix.interfaces.rucio_summoner import RucioSummoner
admix/interfaces/rucio_summoner.py:18: in <module>
    from admix.interfaces.rucio_api import RucioAPI
admix/interfaces/rucio_api.py:21: in <module>
    from rucio.client.client import Client
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/__init__.py:20: in <module>
    from .client import Client  # noqa: F401
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/client.py:30: in <module>
    from rucio.client.accountclient import AccountClient
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/accountclient.py:39: in <module>
    from rucio.client.baseclient import BaseClient
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:92: in <module>
    LOG = setup_logger(module_name=__name__)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/utils.py:1369: in setup_logger
    if _force_cfg_log_level(cfg_opt):
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/utils.py:1345: in _force_cfg_log_level
    cfg_forced_modules = config_get('logging', cfg_option, raise_exception=False, default=None, clean_cached=True)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:53: in config_get
    return get_config().get(section, option)
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:214: in get_config
    __CONFIG = Config()
/home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/common/config.py:238: in __init__
    raise RuntimeError('Could not load Rucio configuration file. '
E   RuntimeError: Could not load Rucio configuration file. Rucio looked in the following paths for a configuration file, in order:
E       /opt/rucio/etc/rucio.cfg
============================================================================================================= warnings summary ==============================================================================================================
../../../../../../../../home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:45
  /home/joran/miniconda3/envs/strax/lib/python3.8/site-packages/rucio/client/baseclient.py:45: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/warnings.html
========================================================================================================== short test summary info ==========================================================================================================
ERROR tests/test_admix.py - RuntimeError: Could not load Rucio configuration file. Rucio looked in the following paths for a configuration file, in order:
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
======================================================================================================== 1 warning, 1 error in 0.56s ========================================================================================================
```

--- a/.gitignore
+++ b/.gitignore
@@ -63,3 +63,4 @@ target/
 
 *~
 bin
+.idea

--- a/setup.py
+++ b/setup.py
@@ -24,12 +24,10 @@ test_requirements = [
 ]
 
 setup(
-    name='admix',
+    name='xe-admix',
     version='0.2.0',
     description="advanced Data Management In Xenon (aDMIX)",
     long_description=readme + '\n\n' + history,
-    author="Boris Bauermeister",
-    author_email='Boris.Bauermeister@gmail.com',
     url='https://github.com/XENON1T/admix',
     packages=find_packages(include=['admix',
                                     'admix.interfaces',

--- a/tests/test_admix.py
+++ b/tests/test_admix.py
@@ -3,32 +3,5 @@
 
 """Tests for `admix` package."""
 
-
-import unittest
-from click.testing import CliRunner
-
-from admix import admix
-from admix import cli
-
-
-class TestAdmix(unittest.TestCase):
-    """Tests for `admix` package."""
-
-    def setUp(self):
-        """Set up test fixtures, if any."""
-
-    def tearDown(self):
-        """Tear down test fixtures, if any."""
-
-    def test_000_something(self):
-        """Test something."""
-
-    def test_command_line_interface(self):
-        """Test the CLI."""
-        runner = CliRunner()
-        result = runner.invoke(cli.main)
-        assert result.exit_code == 0
-        assert 'admix.cli.main' in result.output
-        help_result = runner.invoke(cli.main, ['--help'])
-        assert help_result.exit_code == 0
-        assert '--help  Show this message and exit.' in help_result.output
+def test_admix():
+    import admix

